task/python-boto3/0.1/python-boto3.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: python-boto3
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: CLI
+    tekton.dev/tags: CLI, boto3, sdk
+    tekton.dev/displayName: "python boto3 aws"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    Executes a Python script using AWS Boto3 to manage and inspect AWS resources.
+  params:
+    - name: aws-region
+      type: string
+      default: "us-east-1"
+      description: AWS Region
+  volumes:
+    - name: python-script
+      configMap:
+        name: python-script-configmap
+  steps:
+    - name: run-python-script
+      image: python:3.9
```

Comment:
can we use a full qualified image path along with sha checksum ?

Reply:
done